host/tests/integration/components/catalog-entry-editor-test.gts

```diff
@@ -53,6 +60,8 @@ module('Integration | catalog-entry-editor', function (hooks) {
 
     realm = TestRealm.createWithAdapter(adapter);
     Loader.addRealmFetchOverride(realm);
```

Comment:
instead of creating a loader and configuring it here--configure the loader in the loader service